includes/Framework/Plugin.php

```diff
@@ -363,7 +363,7 @@ public function plugin_action_links( $actions ) {
 		$custom_actions = array();
 
 		// settings url(s)
-		if ( $this->get_square_onboarding_link() ) {
+		if ( $this->get_square_onboarding_link() && 0 === count( wc_square()->get_dependency_handler()->get_missing_php_extensions() ) ) {
```

Comment:
Since we're doing this same thing a few times (getting a count of missing dependencies) wonder if we should add a new method that returns this count? Something like `$this->get_plugin()->get_dependency_handler()->get_number_of_missing_php_extensions()` and can then check if that equals zero or is greater than zero, as needed

Reply:
Added a check to test both missing PHP dependencies and functions:

```
public function meets_php_dependencies() {
	return empty( $this->get_missing_php_extensions() ) && empty( $this->get_missing_php_functions() );
}
```